src/object/obj_class.c

```diff
@@ -276,39 +276,219 @@ static struct daos_obj_class daos_obj_classes[] = {
 		},
 	},
 	{
-		.oc_name	= "EC_2P1G1",
-		.oc_id		= OC_EC_2P1G1,
+		.oc_name	= "EC_S2P1G1",
+		.oc_id		= OC_EC_S2P1G1,
 		{
 			.ca_schema		= DAOS_OS_SINGLE,
 			.ca_resil		= DAOS_RES_EC,
 			.ca_grp_nr		= 1,
 			.ca_ec_k		= 2,
 			.ca_ec_p		= 1,
-			.ca_ec_cell		= 1 << 15,
+			.ca_ec_cell		= 1 << 14,
```

Comment:
not sure if 16K is small cell, maybe can define something like DAOS_EC_CELL_SMALL/_MEDIUM/_LARGE macro then it is convenient to change later.